src/policy/vmspace.rs

```diff
@@ -275,6 +275,17 @@ impl<VM: VMBinding> VMSpace<VM> {
         );
         debug_assert!(self.in_space(object));
         if self.mark_state.test_and_mark::<VM>(object) {
+            // Flip the per-object unlogged bits to "unlogged" state for objects inside the
+            // bootimage
+            #[cfg(feature = "set_unlog_bits_vm_space")]
```

Comment:
Perhaps we can remove the conditional compile feature here since it is likely objects in the VM space will want to have the unlog bit set